tests/integration/test_logcollector/test_configuration/test_basic_configuration_log_format.py

```diff
@@ -274,7 +274,7 @@ def check_log_format_invalid(cfg):
                             error_message=gc.GENERIC_CALLBACK_ERROR_MESSAGE)
 
     if sys.platform != 'win32':
-        log_callback = gc.callback_error_in_configuration('ERROR', prefix,
+        log_callback = gc.callback_error_in_configuration('CRITICAL', prefix,
```

Comment:
Check this change: https://github.com/wazuh/wazuh-qa/pull/4094

Reply:
This was applied after Jenkins instances for this Issue failed on this tests. It was also failing on local executions.